src/lib/FileChunkReference.h

```diff
@@ -27,22 +27,25 @@ enum FileChunkReferenceSize
 class FileChunkReference
 {
 public:
-  FileChunkReference(enum FileChunkReferenceSize fcr_size);
+  FileChunkReference(const enum FileChunkReferenceSize fcr_size);
+  FileChunkReference(const uint64_t stp, const uint64_t cb, const enum FileChunkReferenceSize fcr_size = Size64x64);
```

Comment:
I wonder if it is a better idea to have `fcr_invalid` as the default value of `fcr_size` for this constructor, then thrown an `assert(fcr_size != fcr_invalid)` at the beginning of the parse function, to avoid parsing unwanted integer sizes.. Or maybe I'm too paranoiac

Reply:
I skipped the assert because the switch will prevent reading data anyway. Or would you like to keep it for current debugging purposes?

Reply:
We could actually replace the `ONE_DEBUG_MSG` call with an `assert(false)` in that case. I think this has value for debugging purposes and makes debug output less verbose while "more strict" (as in we fall on the assert).

Could save a few minutes of debugging, instead of having "not good!" hidden somewhere in the debug logs :)

Reply:
I wonder if that'd break stuff at this present moment, if it does then I won't insist!

Reply:
I did not notice a break at this point.